test/e2e/adminUI/tests/group003List/uiTestListView.js

```diff
@@ -13,117 +15,99 @@ module.exports = {
 		browser.adminUISignin.signin();
 
 		browser.adminUIApp.waitForHomeScreen();
+
+		browser.adminUIApp.click('@accessMenu').waitForListScreen();
```

Comment:
We're trying to avoid calls like this? Shouldn't this be along the lines of `browser.adminUIApp.openAccessMenu`?